test008.c: 函数 `f' :
test008.c:9: エラー: `f' の函数定義が重複しています.
test008.c:4: エラー: ここに函数定義がありました.
